sympy/solvers/tests/test_solvers.py

```diff
@@ -747,6 +747,34 @@ def test_issue_5767():
     assert set(solve([x**2 + y + 4], [x])) == \
         set([(-sqrt(-y - 4),), (sqrt(-y - 4),)])
 
+def test_issue_8828():
+    x1 = 0
+    y1 = -620
+    r1 = 920
+    x2 = 126
+    y2 = 276
+    x3 = 51
+    y3 = 205
+    r3 = 104
+
+    f1 = (x-x1)**2 + (y-y1)**2 - (r1-z)**2
```

Comment:
Nice! What a relief to see this fixed. :-)

Could you just apply a little PEP8 spacing here: we try to put space around `+ and -` and put a space after commas.